packages/fast-components-react-base/src/button/README.md

```diff
@@ -1,2 +1,5 @@
 # Button
-The *Button* component can be either an [anchor](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) or [button](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) element but will appear with button styling.
+*Buttons* are commonly used for final actions that complete a user work flow and the text should communicate what will happen after interaction. Typical use cases include submitting a form, closing a [dialog](/components/dialog/), or saving settings. The *button* component can be either an [anchor](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) or [button](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/button) element, but will appear with button styling.
```

Comment:
The base component doesn't assert any styling requirements so I think you should remove the last half of the last sentence.

Reply:
Sure makes sense - removed.